src/main/java/core/basesyntax/shape/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class RightTriangle extends Shape {
+    private double base1;
```

Comment:
Rename 'base1' to 'leg1' for better clarity as right triangles have legs, not bases.